src/MainUtils.cpp

```diff
@@ -88,6 +88,7 @@ void registerDialects() {
 
 void addONNXToMLIRPasses(mlir::PassManager &pm) {
   pm.addPass(mlir::createDecomposeONNXToONNXPass());
+  pm.addPass(mlir::createConstPropONNXToONNXPass());
```

Comment:
Might read better without the "ONNXToONNX" part.

Reply:
Do you want me to remove the ONNXToONNX for both? or change it to O2O?

Reply:
Consider "ConstantFoldingPass"